Source/ACE.sln

```diff
@@ -45,44 +45,74 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ACE.Adapter", "ACE.Adapter\
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
Not sure if this is needed for FakeClient, but for sure non of the already existing projects/sln should have AnyCPU build choices as those were specifically removed a year+ ago to force x64 only

Reply:
I'll fix this by removing the extra build configurations, good call